src/services/redisSessionStorage.ts

```diff
@@ -463,4 +478,34 @@ export default class RedisSessionStorage extends RedisStorageUtils
     }
     return right(replay);
   }
+
+  private parseUserSessionList(
+    userSessionTokensResult: ReadonlyArray<string>
+  ): SessionsList {
+    return userSessionTokensResult.reduce(
+      (prev: SessionsList, _) => {
+        try {
+          const sessionInfoPayload = JSON.parse(_);
```

Comment:
consider using `parseJSON` which returns an Either and cannot throw:

https://github.com/gcanti/fp-ts/blob/ea4643ebc243be824183a61b4a78f8a2651ad49b/docs/modules/Either.ts.md#parsejson-function

Reply:
gotcha ! keep the original version then (or maybe it's worth to backport parseJSON without updating fp-ts ?)

Reply:
I'll try to backport this function ;)

Reply:
Fixed in 2e3769f